commands/bot/roleinfo.js

```diff
@@ -4,7 +4,7 @@ const { getRole } = require("../../config")
 
 module.exports = {
     name: "roleinfo",
-    run: async (message, args, client) => {
+    run: async (message, args) => {
```

Comment:
You can't remove extra parameters. It has to stay (message, args, client). Removing them doesn't even optimize anything, since the variables are created in the messageCreate event and only passed through to the command

Reply:
it worked in help by removing, and I see in some commands they are not there. also why pass if not needed?